products/green_fast_track/models/product/source/_source_models.yml

```diff
@@ -57,5 +57,4 @@ models:
 
   - name: source__shadow_hist_resources_points
   - name: source__shadow_hist_resources_lots
-  - name: source__shadow_hist_resources_polys
```

Comment:
I take it there are no polys after this? The two datasets going into historical_resources are just points and joined lots

Reply:
indeed, just like `historic_resources` and `historic_resources_adj`. double checking right now

Reply:
added a screenshot above shoring that is all points now